test/test_matmul_cuda.py

```diff
@@ -1290,7 +1290,7 @@ def test_float8_error_messages(self, device) -> None:
                 x_fp8,
                 y_fp8,
                 scale_a=torch.ones((M), device="cuda"),
-                scale_b=torch.ones((N, N, 1), device="cuda"),
```

Comment:
calling  transpose on second arg would have failed but the first desired scaling would early return, so this just makes it check the scale_a first since we now early tranpose

Reply:
I specifically modified this check to ensure that we return clear error messages in case of non-2d tensors, instead of cryptic internal assert failures. Could you rework the code and reenable this test?